Update the docs for Integrations and Production Branch (#1981)
vercel/docs: pages/docs/v2/git-integrations/index.mdx

  description: `How to use the ${PRODUCT_NAME} Git Integrations for GitHub, GitLab, or Bitbucket to automatic deploy from your project's repository for each push and ship to production on merge to the main branch.`,
  image: `${process.env.ASSETS}/docs/git-integrations/git-push.png`,
  editUrl: 'pages/docs/v2/git-integrations/index.mdx',
  lastEdited: '2020-07-13T16:54:06.000Z'
}

> ​​<ProductName /> provides integrations for your [GitHub](/docs/v2/git-integrations/vercel-for-github), [GitLab](/docs/v2/git-integrations/vercel-for-gitlab), and [Bitbucket](/docs/v2/git-integrations/vercel-for-bitbucket) projects, allowing for **automatic deployments on every branch push** and merge to the production branch.

Reviewer: ```suggestion
​​<ProductName /> provides integrations for your [GitHub](/docs/v2/git-integrations/vercel-for-github), [GitLab](/docs/v2/git-integrations/vercel-for-gitlab), and [Bitbucket](/docs/v2/git-integrations/vercel-for-bitbucket) projects, allowing for **automatic deployments on every branch push** and merge to the [Production Branch](#production-branch).
```

The <ProductName /> Git Integrations provide the following benefits:

- [Preview Deployments](/docs/v2/platform/deployments#preview) for every push.
- [Production Deployments](/docs/v2/platform/deployments#production) for the most recent changes from the [Production Branch](#production-branch).
- Instant rollbacks when reverting changes assigned to a custom domain.

The easiest way to use a Git Integration is to think of your **`main` branch as production**. Every time a pull/merge request is made to that branch, <ProductName /> will **create a unique deployment**, allowing you to **view the changes in a preview environment before merging**.

In this case, `main` is the Production Branch. When merging to the Production Branch, a **Production Deployment** is made, making the latest changes available to assigned custom domains automatically.

> You can choose to use another branch, or the default branch as the [Production Branch](#production-branch).

Reviewer: ```suggestion
You can choose to use a different branch as the [Production Branch](#production-branch).
```

## Quick Installation

> You can install any Git Integration directly from your dashboard by clicking ["Import Project"](/import) followed by ["Import Git Repository"](/import/git). Then enter the URL of your GitHub, GitLab, or Bitbucket repository to connect and import.

Reviewer: ```suggestion
You can install any Git Integration directly from your dashboard by clicking ["Import Project"](/import) followed by ["Import Git Repository"](/import/git). Then enter the URL of your GitHub, GitLab, or Bitbucket repository to connect and import it.
```

Commit authors will automatically be considered a part of the Team on <ProductName /> if one of the existing members connected their Personal Account on <ProductName /> with the Git account that created the commit.

## Production Branch

> A [Production Deployment](/docs/v2/platform/deployments#production) will be created each time you merge to the **Production Branch**. When you connect a new repository to <ProductName />, `main` will be the Production Branch. If `main` doesn't exist, the default branch will be the Production Branch.

Reviewer: ```suggestion
A [Production Deployment](/docs/v2/platform/deployments#production) will be created each time you merge to the **Production Branch**. When you create a new Project from a Git repository on <ProductName />, `main` will be the Production Branch. If `main` doesn't exist, the Git repository's default branch will be the Production Branch.
```

Reviewer: I understand that it also happens when connecting, but I'd rather be clear on creating a new Project than mixing creating new Projects and connecting existing ones in one sentence.

> A [Production Deployment](/docs/v2/platform/deployments#production) will be created each time you merge to the **Production Branch**. When you create a new Project from a Git repository on <ProductName />, `main` will be the Production Branch. If `main` doesn't exist, the Git repository's default branch will be the Production Branch.

Reviewer: ```suggestion
A [Production Deployment](/docs/v2/platform/deployments#production) will be created each time you merge to the **Production Branch**.

When you create a new Project from a Git repository on <ProductName />, `main` will be the Production Branch. In the case that `main` doesn't exist, the Git repository's default branch will be the Production Branch.
```

> You can change the configuration to Repository Default, `main`, or any other custom branch from the Git Integrations tab in your Project Settings page.

Reviewer 2: ```suggestion
You can change the configuration to Repository Default, `main`, or any other custom branch from the Git Integrations section in your Project Settings page.
```

> You can change the configuration to Repository Default, `main`, or any other custom branch from the Git Integrations section in your Project Settings page.

Reviewer: ```suggestion
On the **Git Integration** page in the **Project Settings**, you can change your Production Branch to be `main`, the Git repository's default branch or any other custom branch of your choice.
```

> On the **Git Integration** page in the **Project Settings**, you can change your Production Branch to be `main`, the Git repository's default branch or any other custom branch of your choice.

Reviewer: ```suggestion
On the **Git Integration** page in the **Project Settings**, you can change your Production Branch to be one of the following:

- The `main` branch.
- The Git repository's default branch.
- Any other custom branch of your choice.

Whenever a new commit is then pushed to the branch you configured here, a [Production Deployment](/docs/v2/platform/deployments#production) will be created for you.
```

<Image
  src={`${process.env.ASSETS}/docs/common/production-branch-configuration.png`}
  width={1498/2}
  height={572/2}
  shadow
  borderRadius
/>

> <Caption>Configuring the Production Branch of your project.</Caption>

Reviewer: ```suggestion
<Caption>Configuring the Production Branch of your Project.</Caption>
```

Commit authors will automatically be considered a part of the Team on <ProductName /> if one of the existing members connected their Personal Account on <ProductName /> with the Git account that created the commit.

## Production Branch

A [Production Deployment](/docs/v2/platform/deployments#production) will be created each time you merge to the **Production Branch**.

### Default Configuration

When you create a new Project from a Git repository on <ProductName />, the Production Branch will be selected in the following order:

- The `main` branch.
- If not present, the `master` branch ([more details](https://vercel.com/blog/production-branch-setting#a-note-on-the-master-branch)).
- If not present, the Git repository's default branch.

> ### Customizing It

Reviewer 3: ```suggestion
### Customizing the Production Branch
```

- The `main` branch.
- If not present, the `master` branch ([more details](https://vercel.com/blog/production-branch-setting#a-note-on-the-master-branch)).

> - If not present, the Git repository's default branch (for Bitbucket, the "Production Branch" setting on the Git repository is used if it's set).

Reviewer 4: Something feels wrong here 🤔 

The sentence starts with `If not present` and then ends with `if it's set`.

What happens if the "Production Branch" is not set?

Reviewer: Clarified!
